client/src/Components/Card/DiagnosticCard.js

```diff
@@ -1,38 +1,27 @@
 import React, { Component } from "react";
-import { graphql } from "react-apollo";
-import { getDiagnosticQuery } from "../../queris/queris";
 import { Card, Icon } from "semantic-ui-react";
-import LoadingSpinner from '../LoadingSpinner';
-
-import DiagnosticDetailCard from "./DiagnosticDetailCard";
 
 
 class DiagnosticCard extends Component {
-  displayCarData() {
-    let data = this.props.data;
-    if (data.loading) {
-      return <LoadingSpinner />;
-    } else {
-      return <div>{data.car.diagnostic}</div>
-    }
-  }
+
   render() {
     return (
       <Card className="data-cards-r1" >
         <Card.Content>
           <Icon circular inverted color="red" name="wrench" size="big" />
           <div className="data-content">
             <p>Diagnostic Issues    </p>
-            <h2>{this.displayCarData()}</h2>
+            <h2>{this.props.diagnostic}</h2>
```

Comment:
prop name is a bit unclear

Reply:
I pushed the latest queryRefactor and client/src/Components/Card/DiagnosticCard.js should be following... 

`    return (
      <Card className="data-cards-r1" >
        <Card.Content>
          <Icon circular inverted color="red" name="wrench" size="big" />
          <div className="data-content">
            <p>Diagnostic Issues    </p>
            <h2>{this.props.diagnostic}</h2>
          </div>
          <Card.Content extra>
            <hr></hr>
            <p>{this.props.diagnosticdetail}</p>

          </Card.Content>
        </Card.Content>
      </Card>
    );`